gammapy/visualization/utils.py

```diff
@@ -20,6 +22,63 @@
 }
 
 
+def plot_rgb(map_, axis="energy", ax=None, **kwargs):
```

Comment:
I think the `ax` keyword should come first as in all our other plot methods.

Maybe remove the `axis` keyword completely? The convention could be that RGB plotting can only have one non-spatial axis with exactly three bins.

And maybe rename the function to `plot_map_rgb()`